Use golang/protobuf instead of gogo/protobuf

ebroberson opened the pull request:
- [X] Read the [Contributing document](../blob/-/.github/CONTRIBUTING.md).

Summary
---------------
[gogo/protobuf](https://github.com/gogo/protobuf) has been deprecated for years.  We should be using up-to-date protobufs provided by [google.golang.org/protobuf](https://github.com/protocolbuffers/protobuf-go).  In order to fit our use cases, a protobuf plugin (`protoc-gen-go-bbs`) has been created to catch any customizations we want to add.

Submodule Pull Requests (please do these first)
---------------
https://github.com/cloudfoundry/auctioneer/pull/20
https://github.com/cloudfoundry/bbs/pull/109
https://github.com/cloudfoundry/cfdot/pull/14
https://github.com/cloudfoundry/executor/pull/116
https://github.com/cloudfoundry/inigo/pull/50
https://github.com/cloudfoundry/locket/pull/27
https://github.com/cloudfoundry/rep/pull/67
https://github.com/cloudfoundry/route-emitter/pull/50
https://github.com/cloudfoundry/routing-api/pull/69
https://github.com/cloudfoundry/routing-info/pull/8
https://github.com/cloudfoundry/vizzini/pull/15

Backward Compatibility
---------------
Breaking Change? **No**

ebroberson committed:
Updates to match new locket protobufs

ebroberson committed:
Remove gogoprotobuf in favor of google protobuf

ebroberson committed:
Updates to use new bbs and locket protobufs

ebroberson committed:
Update to use the latest protobufs

ebroberson committed:
Update struct references to match new protobufs

ebroberson committed:
Updates to use new protobufs

ebroberson committed:
Updates to use new protobufs

ebroberson committed:
Instances has to be set explicitly to 0 since it's a pointer now

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Change port logic to use uint16 instead of int

geofffranks committed:
Update inigo for routing-api port type changes

geofffranks committed:
Fix bad fmting

geofffranks committed:
Fix bug

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
InternalRoutesFromRoutingInfo now takes a pointer

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Clean go.mod
go mod tidy && go mod vendor
Bump bbs (merge issues)
Fix issues from linters

tas-runtime-bot committed:
Bump https://github.com/jqlang/jq/releases/tag/jq-1.7

tas-runtime-bot committed:
Bump https://github.com/rprichard/winpty/releases/tag/0.4.2

ebroberson committed:
Bump bbs (remove old protobuf models)
Bump executor (staticcheck SetNproc)
Remove golang 1.22

tas-runtime-bot committed:
bump credhub-cli

Submodule src/code.cloudfoundry.org/credhub-cli 329c0d63b..5af5e1820:
  > Bump go modules

tas-runtime-bot committed:
Update go.mod dependencies

--------

index d7b4323ea..c7626ce38 100644
@@ -18,31 +18,31 @@ replace (

 	code.cloudfoundry.org/archiver v0.26.0
-	code.cloudfoundry.org/bytefmt v0.28.0
+	code.cloudfoundry.org/bytefmt v0.29.0
 	code.cloudfoundry.org/cacheddownloader v0.0.0-20241210011823-7ae5910b9f48
-	code.cloudfoundry.org/certsplitter v0.38.0
-	code.cloudfoundry.org/cf-routing-test-helpers v0.0.0-20241025163157-ce30ff0fff6d
+	code.cloudfoundry.org/certsplitter v0.39.0
+	code.cloudfoundry.org/cf-routing-test-helpers v0.0.0-20250205194842-92b089ca98e3
 	code.cloudfoundry.org/cf-tcp-router v0.0.0-20241025163552-3216bbbc1656
-	code.cloudfoundry.org/cfhttp/v2 v2.33.0
+	code.cloudfoundry.org/cfhttp/v2 v2.34.0
 	code.cloudfoundry.org/clock v1.28.0
 	code.cloudfoundry.org/cnbapplifecycle v0.0.5
-	code.cloudfoundry.org/credhub-cli v0.0.0-20250203140928-329c0d63b022
-	code.cloudfoundry.org/debugserver v0.36.0
-	code.cloudfoundry.org/diego-logging-client v0.41.0
-	code.cloudfoundry.org/dockerdriver v0.36.0
-	code.cloudfoundry.org/durationjson v0.29.0
-	code.cloudfoundry.org/eventhub v0.28.0
+	code.cloudfoundry.org/credhub-cli v0.0.0-20250210140307-5af5e1820968
+	code.cloudfoundry.org/debugserver v0.37.0
+	code.cloudfoundry.org/diego-logging-client v0.42.0
+	code.cloudfoundry.org/dockerdriver v0.37.0
+	code.cloudfoundry.org/durationjson v0.30.0
+	code.cloudfoundry.org/eventhub v0.29.0
 	code.cloudfoundry.org/garden v0.0.0-20250204164024-2360009b9575
 	code.cloudfoundry.org/go-loggregator/v9 v9.2.1
-	code.cloudfoundry.org/goshims v0.59.0
-	code.cloudfoundry.org/guardian v0.0.0-20250203212249-209fc11dcd6b
-	code.cloudfoundry.org/lager/v3 v3.25.0
-	code.cloudfoundry.org/localip v0.29.0
-	code.cloudfoundry.org/tlsconfig v0.17.0
+	code.cloudfoundry.org/goshims v0.60.0
+	code.cloudfoundry.org/guardian v0.0.0-20250205022619-e7a0755c4c24
+	code.cloudfoundry.org/lager/v3 v3.26.0
+	code.cloudfoundry.org/localip v0.30.0
+	code.cloudfoundry.org/tlsconfig v0.18.0
 	github.com/GaryBoone/GoStats v0.0.0-20130122001700-1993eafbef57
 	github.com/ajstarks/svgo v0.0.0-20211024235047-1546f124cd8b
 	github.com/aws/aws-sdk-go v1.55.6
-	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20250127194757-14dfc70fe71c
+	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20250206222449-6540026091fe
 	github.com/cactus/go-statsd-client v3.1.1-0.20161031215955-d8eabe07bc70+incompatible
 	github.com/cespare/xxhash/v2 v2.3.0
 	github.com/cloudfoundry-community/go-uaa v0.3.3
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-sql-driver/mysql v1.8.1
 	github.com/go-test/deep v1.1.1
-	github.com/gogo/protobuf v1.3.2
 	github.com/golang-jwt/jwt/v4 v4.5.1
 	github.com/golang/protobuf v1.5.4
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/moby/term v0.5.2
 	github.com/nats-io/nats-server/v2 v2.10.25
-	github.com/nats-io/nats.go v1.38.0
+	github.com/nats-io/nats.go v1.39.0
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 	github.com/onsi/ginkgo/v2 v2.22.2
 	github.com/onsi/gomega v1.36.2
 	github.com/tedsuo/ifrit v0.0.0-20230516164442-7862c310ad26
 	github.com/tedsuo/rata v1.0.0
 	github.com/vito/go-sse v1.1.2
-	golang.org/x/crypto v0.32.0
-	golang.org/x/net v0.34.0
-	golang.org/x/oauth2 v0.25.0
+	golang.org/x/crypto v0.33.0
+	golang.org/x/net v0.35.0
+	golang.org/x/oauth2 v0.26.0
 	golang.org/x/sys v0.30.0
-	golang.org/x/time v0.9.0
+	golang.org/x/time v0.10.0
 	google.golang.org/grpc v1.70.0
-	google.golang.org/protobuf v1.36.4
+	google.golang.org/protobuf v1.36.5
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1

tas-runtime-bot committed:
Sync package specs

ebroberson committed:
Bump bbs, executor, rep, garden, grootfs, guardian, idmapper (file-based
service bindings)

ebroberson committed:
Merge branch 'develop' into protobuf-with-plugin

ebroberson committed:
Bump bbs

ebroberson committed:
Fix .final_builds yaml for golang

ebroberson committed:
Add credhub-cli branch to .gitmodules

ebroberson committed:
Bump locket, executor
go mod tidy && go mod vendor

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

geofffranks commented:
I raised a couple questions, and fund a number of spots that could use either cleanup, or uncommenting depending on if the code is actually needed anymore. 

I don't fully understand all the reasons for protoc-gen-bbs, and trying to understand how it's templating the generated code is breaking my brain. 

This is very impressive, well done!

geofffranks commented on .gitmodules:
```diff
@@ -5,39 +5,39 @@
 	path = src/code.cloudfoundry.org/bbs
 	url = https://github.com/cloudfoundry/bbs
-	branch = main
+	branch = protobuf-with-plugin
```
These should all get updated back to main before merging

geofffranks commented on .final_builds/packages/golang-1.23-windows/index.yml:
```diff
@@ -1,12 +1,8 @@
   03d4b9d66556f7eeb02cfd48285a40dc33cd3258a921334e71727950d78ed13c:
     version: 03d4b9d66556f7eeb02cfd48285a40dc33cd3258a921334e71727950d78ed13c
-    blobstore_id: babed125-bfe1-475d-7f0f-dd1016ba7df3
+    blobstore_id: 0179c598-a538-47db-7a7f-81c57c3c5429
```
changes to all the .final_builds/packages concern me here. We should keep whatever's on main rather than the protobuf branch, right?

geofffranks commented on config/blobs.yml:
```diff
@@ -1,7 +1,7 @@
+jq/jq-1.7-linux-amd64.tgz:
+  size: 992916
+  object_id: afe4a362-e7c4-4f06-7f51-b960a496449f
+  sha: sha256:3ccf94a8b783c007c41faf9f20970e2c330c4b705e0c1bf51f9d211a7c5c8dda
```
same concerns for these updates as i had for the .final_builds/packages updates

geofffranks commented on jobs/rep_windows/spec:
```diff
@@ -45,6 +45,9 @@ properties:
       - windows2012R2:/tmp/windows2012R2
+  diego.rep.sidecar_rootfs_path:
+    description: "absolute_path representing the root filesystem used for sidecar processes (ie. "/var/vcap/packages/cflinuxfs4/rootfs.tar"). Must be one of the preloaded_rootfses paths. Leaving the default empty string is ok, as it will then select the first of he preloaded_rootfses"
+    default: ""
```
Is this something you added, or something we removed from the main branch that is about to get re-added?

geofffranks commented:
I think this PR needs rebasing off of origin/develop? It seems like stuff it shouldn't need be changing would end up being changed on the develop branch (package definitions, blob info, (re-?)adding a random job property). Otherwise 👍

ebroberson committed:
Attempt to remove SidecarRootFS changes

ebroberson committed:
Bump credhub-cli, executor, rep, grootfs

ebroberson committed:
Merge branch 'develop' into protobuf-with-plugin

ebroberson committed:
Bump inigo (force-undo revert)

ebroberson committed:
Bump bbs (performance tests)

ebroberson committed:
Bump bbs, locket (new backwards-compatible locket protobufs)

ebroberson committed:
Sync locket spec

ebroberson committed:
Bump bbs, locket

ebroberson committed:
Update for new locket protobufs

ebroberson committed:
Update for new locket protobufs

ebroberson committed:
Update for new locket protobufs

ebroberson committed:
Bump auctioneer, cfdot, rep, route-emitter, routing-api

ebroberson committed:
Bump bbs, locket (regen protos)

ebroberson committed:
Bump locket

ebroberson committed:
Bump bbs, locket

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Bump actioneer, bbs, buildpackapplifecycle, cacheddownloader, cfdot, credhub-cli, executor, inigo, locket, rep, garden, groofs, guardian, idmapper

ebroberson committed:
Merge develop into protobuf-with-plugin

ebroberson committed:
Bump bbs

ebroberson committed:
Bump bbs, locket

tas-runtime-bot committed:
bump credhub-cli garden grootfs guardian idmapper

Submodule src/code.cloudfoundry.org/credhub-cli c28416987..eaea4c141:
  > Bump go modules
Submodule src/garden 6fe78a102..187e9889c:
  > Update go.mod dependencies
  > Update go.mod dependencies
Submodule src/grootfs 417e52cb6..77087f005:
  > Update go.mod dependencies
  > Update go.mod dependencies
  > Update go.mod dependencies
  > Update go.mod dependencies
Submodule src/guardian e98bcba81..c3e2942ba:
  > Update go.mod dependencies
  > Allow rootfs to be imported as flat dir
  > Update go.mod dependencies
  > Merge pull request #470 from PlamenDoychev/main
  > Update go.mod dependencies
  > Merge pull request #471 from cloudfoundry/fix-gats-for-noble
  > Update go.mod dependencies
Submodule src/idmapper c616ffb55..ae2d922a2:
  > Update go.mod dependencies

tas-runtime-bot committed:
Upgrade golang-1.23-linux (1.23.8)

tas-runtime-bot committed:
Update go.mod dependencies

--------

index eb591ca3f..03438c9fb 100644
@@ -13,27 +13,27 @@ replace (

-	code.cloudfoundry.org/archiver v0.32.0
-	code.cloudfoundry.org/bytefmt v0.36.0
+	code.cloudfoundry.org/archiver v0.33.0
+	code.cloudfoundry.org/bytefmt v0.37.0
 	code.cloudfoundry.org/cacheddownloader v0.0.0-20250312193827-23c030d5e4f3
-	code.cloudfoundry.org/certsplitter v0.47.0
+	code.cloudfoundry.org/certsplitter v0.48.0
 	code.cloudfoundry.org/cf-routing-test-helpers v0.0.0-20250228152907-f87c227cf5e7
 	code.cloudfoundry.org/cf-tcp-router v0.0.0-20250228153011-2de6b290768e
-	code.cloudfoundry.org/cfhttp/v2 v2.41.0
-	code.cloudfoundry.org/clock v1.34.0
+	code.cloudfoundry.org/cfhttp/v2 v2.42.0
+	code.cloudfoundry.org/clock v1.35.0
 	code.cloudfoundry.org/cnbapplifecycle v0.0.5
-	code.cloudfoundry.org/credhub-cli v0.0.0-20250407131255-c28416987f45
-	code.cloudfoundry.org/debugserver v0.45.0
-	code.cloudfoundry.org/diego-logging-client v0.50.0
-	code.cloudfoundry.org/dockerdriver v0.45.0
-	code.cloudfoundry.org/durationjson v0.37.0
-	code.cloudfoundry.org/eventhub v0.36.0
-	code.cloudfoundry.org/garden v0.0.0-20250402022550-6fe78a10209e
+	code.cloudfoundry.org/credhub-cli v0.0.0-20250414180459-eaea4c141b26
+	code.cloudfoundry.org/debugserver v0.46.0
+	code.cloudfoundry.org/diego-logging-client v0.51.0
+	code.cloudfoundry.org/dockerdriver v0.46.0
+	code.cloudfoundry.org/durationjson v0.38.0
+	code.cloudfoundry.org/eventhub v0.37.0
+	code.cloudfoundry.org/garden v0.0.0-20250416022737-187e9889c592
 	code.cloudfoundry.org/go-loggregator/v9 v9.2.1
-	code.cloudfoundry.org/goshims v0.67.0
-	code.cloudfoundry.org/guardian v0.0.0-20250402023119-e98bcba8165d
-	code.cloudfoundry.org/lager/v3 v3.33.0
-	code.cloudfoundry.org/localip v0.36.0
+	code.cloudfoundry.org/goshims v0.68.0
+	code.cloudfoundry.org/guardian v0.0.0-20250416185610-c3e2942ba6ce
+	code.cloudfoundry.org/lager/v3 v3.34.0
+	code.cloudfoundry.org/localip v0.37.0
 	code.cloudfoundry.org/tlsconfig v0.24.0
 	github.com/GaryBoone/GoStats v0.0.0-20130122001700-1993eafbef57
 	github.com/ajstarks/svgo v0.0.0-20211024235047-1546f124cd8b
 	github.com/cespare/xxhash/v2 v2.3.0
 	github.com/cloudfoundry-community/go-uaa v0.3.5
 	github.com/cloudfoundry/dropsonde v1.1.0
-	github.com/containers/image/v5 v5.34.3
+	github.com/containers/image/v5 v5.35.0
 	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7
 	github.com/envoyproxy/go-control-plane v0.13.4
 	github.com/fsnotify/fsnotify v1.9.0
 	github.com/lib/pq v1.10.9
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/moby/term v0.5.2
-	github.com/nats-io/nats-server/v2 v2.11.0
-	github.com/nats-io/nats.go v1.41.0
+	github.com/nats-io/nats-server/v2 v2.11.1
+	github.com/nats-io/nats.go v1.41.1
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 	github.com/onsi/ginkgo/v2 v2.23.4
 	github.com/onsi/gomega v1.37.0

tas-runtime-bot committed:
Upgrade golang-1.23-windows (1.23.8)

tas-runtime-bot committed:
Sync package specs

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Bump bbs, inigo, rep, garden, grootfs, guardian, idmapper

ebroberson committed:
Merge branch 'develop' into protobuf-with-plugin

ebroberson committed:
go mod tidy && go mod vendor

tas-runtime-bot committed:
bump credhub-cli garden grootfs guardian idmapper

Submodule src/code.cloudfoundry.org/credhub-cli eaea4c141..cded072a8:
  > Bump go modules
Submodule src/garden 3dcb20fef..cd1e49205:
  > Update go.mod dependencies
Submodule src/grootfs df71fddc9..1342a6e56:
  > Update go.mod dependencies
Submodule src/guardian d1f3b9fb8..561eda7dd:
  > Update go.mod dependencies
Submodule src/idmapper 708d2700d..b6751f73f:
  > Update go.mod dependencies

tas-runtime-bot committed:
Update go.mod dependencies

--------

index d1ec73762..bec05101b 100644
@@ -13,28 +13,28 @@ replace (

-	code.cloudfoundry.org/archiver v0.33.0
-	code.cloudfoundry.org/bytefmt v0.37.0
+	code.cloudfoundry.org/archiver v0.34.0
+	code.cloudfoundry.org/bytefmt v0.38.0
 	code.cloudfoundry.org/cacheddownloader v0.0.0-20250312193827-23c030d5e4f3
-	code.cloudfoundry.org/certsplitter v0.48.0
+	code.cloudfoundry.org/certsplitter v0.49.0
 	code.cloudfoundry.org/cf-routing-test-helpers v0.0.0-20250228152907-f87c227cf5e7
 	code.cloudfoundry.org/cf-tcp-router v0.0.0-20250228153011-2de6b290768e
-	code.cloudfoundry.org/cfhttp/v2 v2.42.0
-	code.cloudfoundry.org/clock v1.35.0
+	code.cloudfoundry.org/cfhttp/v2 v2.43.0
+	code.cloudfoundry.org/clock v1.36.0
 	code.cloudfoundry.org/cnbapplifecycle v0.0.5
-	code.cloudfoundry.org/credhub-cli v0.0.0-20250414180459-eaea4c141b26
-	code.cloudfoundry.org/debugserver v0.46.0
-	code.cloudfoundry.org/diego-logging-client v0.51.0
-	code.cloudfoundry.org/dockerdriver v0.46.0
-	code.cloudfoundry.org/durationjson v0.38.0
-	code.cloudfoundry.org/eventhub v0.37.0
-	code.cloudfoundry.org/garden v0.0.0-20250416022737-187e9889c592
+	code.cloudfoundry.org/credhub-cli v0.0.0-20250421130851-cded072a83a8
+	code.cloudfoundry.org/debugserver v0.47.0
+	code.cloudfoundry.org/diego-logging-client v0.52.0
+	code.cloudfoundry.org/dockerdriver v0.47.0
+	code.cloudfoundry.org/durationjson v0.39.0
+	code.cloudfoundry.org/eventhub v0.38.0
+	code.cloudfoundry.org/garden v0.0.0-20250421112302-cd1e492051ed
 	code.cloudfoundry.org/go-loggregator/v9 v9.2.1
-	code.cloudfoundry.org/goshims v0.68.0
-	code.cloudfoundry.org/guardian v0.0.0-20250416185610-c3e2942ba6ce
-	code.cloudfoundry.org/lager/v3 v3.34.0
-	code.cloudfoundry.org/localip v0.37.0
-	code.cloudfoundry.org/tlsconfig v0.24.0
+	code.cloudfoundry.org/goshims v0.69.0
+	code.cloudfoundry.org/guardian v0.0.0-20250421112904-561eda7dd718
+	code.cloudfoundry.org/lager/v3 v3.35.0
+	code.cloudfoundry.org/localip v0.38.0
+	code.cloudfoundry.org/tlsconfig v0.25.0
 	github.com/GaryBoone/GoStats v0.0.0-20130122001700-1993eafbef57
 	github.com/ajstarks/svgo v0.0.0-20211024235047-1546f124cd8b
 	github.com/aws/aws-sdk-go v1.55.6
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/moby/term v0.5.2
 	github.com/nats-io/nats-server/v2 v2.11.1
-	github.com/nats-io/nats.go v1.41.1
+	github.com/nats-io/nats.go v1.41.2
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 	github.com/onsi/ginkgo/v2 v2.23.4
 	github.com/onsi/gomega v1.37.0
 	golang.org/x/oauth2 v0.29.0
 	golang.org/x/sys v0.32.0
 	golang.org/x/time v0.11.0
-	google.golang.org/grpc v1.71.1
+	google.golang.org/grpc v1.72.0
 	google.golang.org/protobuf v1.36.6
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1

ebroberson committed:
Bump locket (regen certs)

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Bump+merge submodules

ebroberson committed:
Merge branch 'develop' into protobuf-with-plugin

ebroberson committed:
Bump bbs

tas-runtime-bot committed:
Upgrade golang-1.24-linux (1.24.2)

tas-runtime-bot committed:
Upgrade golang-1.24-windows (1.24.2)

tas-runtime-bot committed:
Upgrade golang-1.24-linux (1.24.3)

tas-runtime-bot committed:
Upgrade golang-1.24-windows (1.24.3)

ebroberson committed:
Bump bbs, locket

ebroberson committed:
Bump bbs (skip perf tests)

ebroberson committed:
Bump bbs (task reuse fix)

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
Merge branch 'main' into protobuf-with-plugin

ebroberson committed:
submodules: merge main

ebroberson committed:
Merge branch 'develop' into protobuf-with-plugin

ebroberson committed:
Sync package specs

ebroberson committed:
Bump bbs, lock (regen protos)

ameowlia committed:
add archive message

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ameowlia committed:
archive notice

ebroberson committed:
Remove submodule auction for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/auction/' from commit '1db576e7bb568d63aa730f09b66058644e9f3b06'

git-subtree-dir: src/code.cloudfoundry.org/auction
git-subtree-mainline: 3797571f71340e69c47b79f66038516a4491a0eb
git-subtree-split: 1db576e7bb568d63aa730f09b66058644e9f3b06

ebroberson committed:
Remove .github and README-type files from auction

ebroberson committed:
Remove submodule auctioneer for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/auctioneer/' from commit '040727f58bfda39ec4d3f1a8c86c5ea71e8d12ac'

git-subtree-dir: src/code.cloudfoundry.org/auctioneer
git-subtree-mainline: c32737b7c43afdb2bad9dfbde7f6695ef8204405
git-subtree-split: 040727f58bfda39ec4d3f1a8c86c5ea71e8d12ac

ebroberson committed:
Remove .github and README-type files from auctioneer

ebroberson committed:
Remove submodule cfdot for subtree migration

ebroberson committed:
Remove .github and README-type files from cfdot

ebroberson committed:
Add 'src/code.cloudfoundry.org/cfdot/' from commit '3559f3d10b6ac3e27f638b5693b1004a3206c1ef'

git-subtree-dir: src/code.cloudfoundry.org/cfdot
git-subtree-mainline: 25f20b34381716ff7b2f3b0c03104759d236c315
git-subtree-split: 3559f3d10b6ac3e27f638b5693b1004a3206c1ef

ebroberson committed:
Remove submodule diego-ssh for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/diego-ssh/' from commit 'c7483e25ab4885b6103098f8524421413063043f'

git-subtree-dir: src/code.cloudfoundry.org/diego-ssh
git-subtree-mainline: ce579e4ae5f497d7cd07c51d9ff079405d37a37d
git-subtree-split: c7483e25ab4885b6103098f8524421413063043f

ebroberson committed:
Remove .github and README-type files from diego-ssh

ebroberson committed:
Remove submodule dockerapplifecycle for subtree migration

ebroberson committed:
Remove .github and README-type files from dockerapplifecycle

ebroberson committed:
Add 'src/code.cloudfoundry.org/dockerapplifecycle/' from commit 'e707e8b0f9bcd975fe60cc8e36190f838820ff40'

git-subtree-dir: src/code.cloudfoundry.org/dockerapplifecycle
git-subtree-mainline: 462c640722f4dfb99867a3ae9bb01678fa1e176e
git-subtree-split: e707e8b0f9bcd975fe60cc8e36190f838820ff40

ebroberson committed:
Remove submodule ecrhelper for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/ecrhelper/' from commit '5bf65e63bab5cdac77a22371e2930a77e5e07bc7'

git-subtree-dir: src/code.cloudfoundry.org/ecrhelper
git-subtree-mainline: df82ffd0700c3532732aec3abcdd48f239b1b0f5
git-subtree-split: 5bf65e63bab5cdac77a22371e2930a77e5e07bc7

ebroberson committed:
Remove .github and README-type files from ecrhelper

ebroberson committed:
Remove submodule fileserver for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/fileserver/' from commit '7140418ad4b83e98626668796d076d1f270a88f3'

git-subtree-dir: src/code.cloudfoundry.org/fileserver
git-subtree-mainline: 4b54a41698ca076c31092d824ef6cd5ccac9555e
git-subtree-split: 7140418ad4b83e98626668796d076d1f270a88f3

ebroberson committed:
Remove .github and README-type files from fileserver

ebroberson committed:
Remove submodule healthcheck for subtree migration

ebroberson committed:
Remove .github and README-type files from healthcheck

ebroberson committed:
Add 'src/code.cloudfoundry.org/healthcheck/' from commit '333a80d1fe900c32fc95d09ae69942b65ac9bf7d'

git-subtree-dir: src/code.cloudfoundry.org/healthcheck
git-subtree-mainline: 149127be22f632b93022771d8e339babccc08f8a
git-subtree-split: 333a80d1fe900c32fc95d09ae69942b65ac9bf7d

ebroberson committed:
Remove submodule inigo for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/inigo/' from commit '67ea2bdbf87d64658b9aefaa059719ffc7c85385'

git-subtree-dir: src/code.cloudfoundry.org/inigo
git-subtree-mainline: 97df0d29bd122f59b0ee52b80f004842de97af80
git-subtree-split: 67ea2bdbf87d64658b9aefaa059719ffc7c85385

ebroberson committed:
Remove .github and README-type files from inigo

ebroberson committed:
Remove submodule localdriver for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/localdriver/' from commit '57e5bd5b1e0846ce4427ebad091817e3ff8e7293'

git-subtree-dir: src/code.cloudfoundry.org/localdriver
git-subtree-mainline: 30926cd4637f43d04fec9e19759eb9e24b517de3
git-subtree-split: 57e5bd5b1e0846ce4427ebad091817e3ff8e7293

ebroberson committed:
Remove .github and README-type files from localdriver

ebroberson committed:
Remove submodule operationq for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/operationq/' from commit 'b1c6a9c2437f29b8bdddb94b848e40bacb4b2754'

git-subtree-dir: src/code.cloudfoundry.org/operationq
git-subtree-mainline: 7dfb773a9e30f4e72014662d50334fb53532e25e
git-subtree-split: b1c6a9c2437f29b8bdddb94b848e40bacb4b2754

ebroberson committed:
Remove .github and README-type files from operationq

ebroberson committed:
Remove submodule route-emitter for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/route-emitter/' from commit '186bee7f23a7814bbcf246df5efc09117940f09b'

git-subtree-dir: src/code.cloudfoundry.org/route-emitter
git-subtree-mainline: 58a5a4d1f6ef8805963cef88cd65c42f3b4ae637
git-subtree-split: 186bee7f23a7814bbcf246df5efc09117940f09b

ebroberson committed:
Remove .github and README-type files from route-emitter

ebroberson committed:
Remove submodule vizzini for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/vizzini/' from commit '4633e53c7b2fd569c093ac9057c3f26e220edbbc'

git-subtree-dir: src/code.cloudfoundry.org/vizzini
git-subtree-mainline: 5f6c688d3b41f4ccf03fc24c810d8bade3ff65bd
git-subtree-split: 4633e53c7b2fd569c093ac9057c3f26e220edbbc

ebroberson committed:
Remove .github and README-type files from vizzini

ebroberson committed:
Remove submodule volman for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/volman/' from commit '1cc72f1031b767fea02a23c8ab11933f6d280b8d'

git-subtree-dir: src/code.cloudfoundry.org/volman
git-subtree-mainline: 2f2a4a53d0b3d3aa4f987b8835e0abba718a1d7a
git-subtree-split: 1cc72f1031b767fea02a23c8ab11933f6d280b8d

ebroberson committed:
Remove .github and README-type files from volman

ebroberson committed:
Remove submodule workpool for subtree migration

ebroberson committed:
Add 'src/code.cloudfoundry.org/workpool/' from commit '1489753f182ea854880ed7803cafecd9e6e870d8'

git-subtree-dir: src/code.cloudfoundry.org/workpool
git-subtree-mainline: 380c9a5a707924ba296908269a99b84edb578d0c
git-subtree-split: 1489753f182ea854880ed7803cafecd9e6e870d8

ebroberson committed:
Remove .github and README-type files from workpool

ebroberson committed:
go mod tidy && go mod vendor

ebroberson committed:
Update submodules

ebroberson committed:
Handle merge conflicts

ebroberson committed:
Bump credhub-cli

ebroberson committed:
go mod tidy && go mod vendor

ebroberson committed:
Manual sync package spec

ebroberson committed:
CI doesn't like that ginkgo/automaxprocs exists in vizzini spec
